ansible/rocketpool.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Verify Ansible versions
+  hosts: all
+  tags: always
+  become: false
+  run_once: true
+  gather_facts: false
+  tasks:
+    - local_action: command ./versioncheck.py
+      changed_when: false
+
+- name: Configure RocketPool node
+  become: true
+  hosts:
+    - nimbus-rayonism-qmerge
+  roles:
+    -
+      role: infra-role-rocketpool-smart-node
+      tags: rocketpool
+      vars:
+        rocketpool_web3_url: 'http://10.13.0.51:8545'
+        rocketpool_web3_ws_url: 'ws://10.13.0.51:8546'
```

Comment:
Why are you setting those here? Just set them in `ansible/group_vars/nimbus.rayonism.yml`.
Also, use `ansible/rayonism.yml` rather than making a new playbook.

Reply:
And use hostnames rather than IPs.

Reply:
```
admin@goerli-01.aws-eu-central-1a.nimbus.geth:~ % grep 10.13.0.51 /etc/hosts
10.13.0.51    goerli-01.aws-eu-central-1a.nimbus.geth.wg
```

Reply:
Fixed